README.md

```diff
@@ -0,0 +1,65 @@
+Vendors can leverage CloudBlue Connect to:
+-   Reduce the total cost of ownership for homegrown technology supporting their indirect channel
+-   Standardize integrations with partners
+-   Increase efficiencies and minimize redundancies by bridging their direct and indirect sales channels
+
+Distributors can use CloudBlue Connect to:
+-   Define, manage and distribute any type of product (omni-product) through any channel (omni-channel)
+-   Transform perpetual licensed products into a subscription model
+-   Onboard new products into their portfolio quickly to build and deliver unique solutions to end customers
+
+Mulesoft Extension for Cloudblue Connect enables you to create apps that react to Cloudblue Connect events such as adding, changing, or deleting objects like purchase requests, assets, accounts.
+
+It allows an easy and fast integration with the CloudBlue Connect Platform. Thanks to it you can automate the fulfillment of orders generated by your products.
+
+Cloudblue Connect Mulesoft Extension exposes methods for accessing Cloudblue Connect and uses the public API defined at [Cloudblue Connect OpenAPI Specification](https://connect.cloudblue.com/community/api/openapi/).
+
+
+This extension has all of the functionallity exposed in the API definition, and provides you with all access to the functionality of the Connect Platform.
+
+## Prerequisites
+To use this connector, you must be familiar with:
+
+-   The Cloudblue Connect operation    
+-   Anypoint Studio    
+-   Mule runtime engine (Mule)   
+-   Elements and global elements in a Mule flow    
+-   Creating a Mule app using Anypoint Studio (Studio)
+
+## Getting Started
+
+In this repository you will find the code to deploy the Cloudblue Connect Mulesoft Extension. Also you can download from Maven repository. 
+Once clonning the repo, depending of the scenario, the instalation is different. 
+
+### Install Connector to use in Anypoint Studio
+Anypoint Studio Allows to build, test and deploy Mulesoft Apps in the Mulesoft Cloud Platform or in on-prem platform. 
+The documentation to deploy the connector in Anypoint studio is available [here](https://connect.cloudblue.com/community/extensions/mulesoft/cloudblue-connect-mulesoft-extension/install-connector-to-use-in-anypoint-studio/).
+
+### Install Connector to use in Design Center
+Once you deploy the connector in Anypoint Exchange Platform, you can share, download or use in the Design Center Platform.
+The documentation to deploy the connector in Anypoint Exchange Platform is available [here](https://connect.cloudblue.com/community/extensions/mulesoft/cloudblue-connect-mulesoft-extension/install-connector-to-use-in-design-center/).
+
+## Build an Mulesoft app using Cloudblue Connect Mulesoft Extension
+In Mulesoft ecosistem, an integration between two or more platforms is called Application or App.
+It's possible to build an App in both platforms using the cloudblue extension. 
```

Comment:
If you have any documentation page with example app, then linking here will be great.

Reply:
No yet.